lang/cs/Org.Apache.REEF.Network/Elastic/Driver/IDriverMessage.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+using Org.Apache.REEF.Utilities.Attributes;
+
+namespace Org.Apache.REEF.Network.Elastic.Driver
+{
+    /// <summary>
+    /// Message sent by the Driver to operators on running Tasks. 
+    /// This message contains instructions from the Driver to Tasks.
+    /// </summary>
+    [Unstable("0.16", "API may change")]
+    public interface IDriverMessage
```

Comment:
We have a class like this already in REEF, which supports Driver to Task messaging. What's the added value of this one?

Reply:
The IDriverMessage in Commons has only a Optional<byte[]> Message field while mine is a bit more structured. Indeed when received by task, I will get a IDriverMessage which I then specialize into ElasticDriverMessageImpl.  

    public void Handle(IDriverMessage value)
    {
            if (value.Message.IsPresent())
            {
                var message = ElasticDriverMessageImpl.From(value.Message.Value);

`